BasicGates/ReferenceImplementation.qs

```diff
@@ -72,27 +72,19 @@ namespace Quantum.Kata.BasicGates {
     //        If the qubit is in state |0⟩, change its state to cos(alpha)*|0⟩ + sin(alpha)*|1⟩.
     //        If the qubit is in state |1⟩, change its state to -sin(alpha)*|0⟩ + cos(alpha)*|1⟩.
     //        If the qubit is in superposition, change its state according to the effect on basis vectors.
-    operation AmplitudeChange_Reference (q : Qubit, alpha : Double) : Unit {
-        
-        body (...) {
-            Ry(2.0 * alpha, q);
-        }
-        
-        adjoint invert;
+    operation AmplitudeChange_Reference (q : Qubit, alpha : Double) : Unit
+    is Adj {        
```

Comment:
I can do that, just keep in mind that 
operation Foo () : ((Unit => Unit) is Adj) {
...
}
is not the same as 
operation Foo () : (Unit => Unit) is Adj {
...
}
To avoid that confusion, the current formatting is to put it on a new line. We can have a chat about that offline.

Reply:
I'll be happy to chat offline on Monday. I think that in the katas the chances for this kind of confusion are extremely rare, and in the `Tasks.qs` files, especially in the BasicGates kata where we don't have any operations returning operations, having `is Adj` instead of `body` has a higher chance to confuse the reader.